auth_saml/models/auth_saml.py

```diff
@@ -0,0 +1,86 @@
+# Copyright (C) 2010-2016 XCG Consulting <http://odoo.consulting>
+# License AGPL-3.0 or later (http://www.gnu.org/licenses/agpl).
+
+import logging
+import json as simplejson
+
+from odoo import api, fields, models
+
+_logger = logging.getLogger(__name__)
+try:
+    import lasso
+except ImportError:
+    _logger.debug('Cannot `import lasso`.')
```

Comment:
```suggestion
import lasso
```

No need to protect imports in v12+.